utils/windows.py

```diff
@@ -4,11 +4,17 @@
 
 
 from loguru import logger
-import win32com.client
-import win32com.client as win32
 
```

Comment:
+import platform

If “import platform” is not added, the following errors will occur:
excels_processor:_combine_product:327 - Error: name 'platform' is not defined
excels_processor:export_data:96 - Export to poc_tool8 failed."